storm-server/src/main/java/org/apache/storm/scheduler/Cluster.java

```diff
@@ -700,17 +703,17 @@ public void assign(SchedulerAssignment assignment, boolean ignoreSingleException
     }
 
     /**
-     * Calculate the amount of shared off heap memory on a given nodes with the given assignment.
+     * Calculate the amount of shared off heap node memory on a given node with the given assignment.
      *
      * @param nodeId     the id of the node
      * @param assignment the current assignment
-     * @return the amount of shared off heap memory for that node in MB
+     * @return the amount of shared off heap node memory for that node in MB
      */
-    private double calculateSharedOffHeapMemory(String nodeId, SchedulerAssignmentImpl assignment) {
-        return calculateSharedOffHeapMemory(nodeId, assignment, null);
+    private double calculateSharedOffHeapNodedMemory(String nodeId, SchedulerAssignmentImpl assignment) {
```

Comment:
Did you mean Nodeid or NodeId?

Reply:
typo: should read calculateSharedOffHeapNodeMemory